java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -522,6 +524,49 @@ public void getdel() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void getex() {
+        // setup
+        String key = "testKey";
+        String value = "testValue";
+        CompletableFuture<String> testResponse = new CompletableFuture<>();
+        testResponse.complete(value);
+        when(commandManager.<String>submitNewCommand(eq(GetEx), eq(new String[] {key}), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<String> response = service.getex(key);
+        String payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(value, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void getex_options() {
+        // setup
+        GetExOptions options = GetExOptions.Seconds(10L);
+        String[] arguments = new String[] {"key", "EX", "10"};
```

Comment:
sorry to say it, but we should have a test for each option variant.  Especially "PERSIST".